src/test/java/org/traccar/protocol/MictrackProtocolDecoderTest.java

```diff
@@ -32,6 +33,17 @@ public void testDecode() throws Exception {
         verifyAttributes(decoder, text(
                 "MT;5;866425031379169;RH;5+190116112648+0+0+0+0+11+3954+1"));
 
+        // Mictrack Low Altitude Flight tests
```

Comment:
If you really want to distinguish, please split into two tests with proper names. Remove comments.